vividus-to-xray-exporter/src/main/java/org/vividus/xray/exporter/XrayExporter.java

```diff
@@ -97,7 +98,10 @@ public void exportResults() throws IOException
             }
         }
 
-        addTestCasesToTestExecution(testCaseIds);
+        consumeOnNotNull(xrayExporterOptions.getTestExecutionKey(),
+            key -> xrayFacade.updateTestExecution(key, testCaseIds));
+        consumeOnNotNull(xrayExporterOptions.getTestSetKey(), key -> xrayFacade.updateTestSet(key, testCaseIds));
```

Comment:
can `testSetKey` be `Optional<String>` in `XrayExporterOptions` ?

Reply:
but even if it can, what is the benefit of using it?
- imho Optional.ofPresent is not good option due to thrown exception
- imho Optional.isPresent and then Optional.get is worse than just simple null check